bip-0077.md

```diff
@@ -0,0 +1,703 @@
+returns it in the response body with status 200 OK.
+
+Upon receiving an encapsulated 200 OK response, the receiver decrypts
+the payload and checks the *Original PSBT* therein according to the
+[receiver checklist](#receivers-original-psbt-checklist).
+
+The receiver then updates the *Original PSBT* to include new signed
+inputs and outputs, invalidating the sender's signature(s). The receiver
+may also adjust the transaction fee. The result, called the *Proposal
+PSBT*, must satisfy the [sender checklist](#senders-proposal-psbt-checklist)
+
+The Proposal PSBT MUST:
+
+- Include complete UTXO data.
+- Include all inputs from the Original PSBT.
+- Include all outputs which do not belong to the receiver from the
+  Original PSBT.
+- Insert additional inputs and outputs at a random index.
+
+The Proposal PSBT sender MAY:
+
+- Add, remove or modify Original PSBT outputs under the control of the
+  receiver (i.e. not sender change).
+
+The Proposal PSBT MUST NOT:
+
+- Shuffle the order of inputs or outputs contained in the Original PSBT.
+- Decrease the absolute fee of the Original PSBT.
+
+The receiver encrypts the *Proposal PSBT* to the sender's reply key according to
+[HPKE Auth
+mode](https://www.rfc-editor.org/rfc/rfc9180.html#name-authentication-using-an-asy),
+using the receiver's key for authentication. The HPKE `info` string is
+`PjV2MsgB`. The HPKE wire format is the same as in the [sender's
+message](#sender-original-psbt-messaging).
+
+```
+PjV2MsgB Byte Representation (7168 bytes total)
++---------------------------------------------------------------------------------------+
+| ElligatorSwift |                             Ciphertext                               |
+|   (64 bytes)   |                            (7104 bytes)                              |
+|                +---------------------------------------------------------+------------+
+|                |           Padded Plaintext                              |  AEAD Tag  |
+|                |       (7088 bytes = 7168-64-16)                         | (16 bytes) |
++---------------------------------------------------------------------------------------+
+```
+
+The receiver makes the resulting HPKE payload the body of a POST request to the
+sender's mailbox whose Short ID is derived from the sender's reply key. This request is then [
+encapsulated](#client-directory-interactions) according to
+Oblivious HTTP to the directory's OHTTP Gateway. OHTTP serializes the
+inner request as BHTTP, and provides another layer of HPKE encryption,
+between the client and directory.
+
+Once the receiver makes this request, they wait for either transaction from the
+Original PSBT or Proposal PSBT to be broadcast to the Bitcoin network.
+
+#### Receiver's Original PSBT checklist
+
+The receiver checklist is the same as the [the BIP 78 receiver
+checklist](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#receivers-original-psbt-checklist).
```

Comment:
I think it’s valuable to point out that it’s the same between BIP 78 and BIP 77, but it might make sense to copy the check list here, given that this is the superseding document.

Reply:
Because BIP 78 is still Draft status, does it not make sense to refer to that potentially evolving document?

Reply:
Our checklist would change if BIP 78 decided to change theirs, however I find it unlikely that any change would be seriously entertained at this point after #1605, #1244, #1396 have already been in.

Reply:
Then it seems fine to keep it as is